.travis.yml

```diff
@@ -9,7 +9,7 @@ install:
   - pip freeze
 
 before_script:
-  - pip install flake8 && flake8 kale example --max-line-length 100
+  - pip install flake8 && flake8 kale example --max-line-length 100 --ignore=E731,E402
```

Comment:
stop putting `__author__ = me` before imports to fix E402.  you are not more important than your imports.  it also reads strangely if you `from __future__`.